recipes/ftxui/all/test_package/CMakeLists.txt

```
@@ -0,0 +1,11 @@
+cmake_minimum_required(VERSION 3.11)
+project(test_package)
+
+SET(CMAKE_CXX_STANDARD 11)
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup(TARGETS)
+
+find_package(ftxui REQUIRED CONFIG)
+add_executable(${PROJECT_NAME} test_package.cpp)
+target_link_libraries(${PROJECT_NAME} CONAN_PKG::ftxui)
```

```suggestion
target_link_libraries(${PROJECT_NAME} ftxui::ftxui)
```
Please use `ftxui::ftxui` like other recipes.